test/library/standard/Set/leaks/doubleStringAdd.execopts

```diff
@@ -0,0 +1 @@
+--memLeaks
```

Comment:
I think you could also potentially make this test use a skipif so it only runs with `--memLeaks`.  I have a vague feeling that there might be some compilation modes that are incompatible with `--memLeaks` but I may be mixing it up with gasnet or asan or something, so it might not be necessary to change this.

Reply:
I seem to be seeing that most similar PRs add an execopt rather than a skipif (like https://github.com/chapel-lang/chapel/pull/18779), but your comment does make me wonder if I am missing something. I was not able to determine how to make a skipif for memory leaks based on https://github.com/chapel-lang/chapel/blob/main/doc/rst/developer/bestPractices/TestSystem.rst and do not see any particular environment variables that are set or anything for that configuration that would be easy to determine in a skipif. 

Is there a simple way to do that that I am missing?

Reply:
Yeah, most of our skipifs are based on environment variables but it is possible to base it off of a passed flag.  Take a look at test/compflags/bradc/minimalModules/SKIPIF for an example.

Though, it is possible that memleaks testing doesn't run futures at all, so that may be the wrong choice (I can never remember).  Whichever way you choose, you definitely want to explicitly go looking for the test in the memleaks Jenkins log the next day